Common/UnitDefinitions/MagneticField.json

```diff
@@ -14,13 +14,25 @@
       "PluralName": "Teslas",
       "FromUnitToBaseFunc": "x",
       "FromBaseToUnitFunc": "x",
-	  "Prefixes": [ "Nano", "Micro", "Milli" ],
+	    "Prefixes": [ "Nano", "Micro", "Milli" ],
       "Localization": [
         {
           "Culture": "en-US",
           "Abbreviations": [ "T" ]
         }
       ]
+    },
+    {
+      "SingularName": "Gauss",
+      "PluralName": "Gausses",
```

Comment:
I found this strange at first, but apparently both Gauss and Gausses are valid plural forms.
https://en.wiktionary.org/wiki/gauss
https://www.merriam-webster.com/dictionary/gauss